docs/source.md

```diff
@@ -4,7 +4,31 @@ Below are instructions for 18.04 LTS. For other Linux and Unix systems, steps ma
 
 Beware of [dependencies](https://github.com/jolibrain/deepdetect/tree/master/docs/dependencies.md), typically on Debian/Ubuntu Linux, do:
 ```
-sudo apt-get install build-essential libgoogle-glog-dev libgflags-dev libeigen3-dev libopencv-dev libcppnetlib-dev libboost-dev libboost-iostreams-dev libcurlpp-dev libcurl4-openssl-dev protobuf-compiler libopenblas-dev libhdf5-dev libprotobuf-dev libleveldb-dev libsnappy-dev liblmdb-dev libutfcpp-dev cmake libgoogle-perftools-dev unzip python-setuptools python-dev libspdlog-dev python-six python-enum34 libarchive-dev rapidjson-dev libmapbox-variant-dev
+sudo apt-get install build-essential libgoogle-glog-dev libgflags-dev libeigen3-dev libopencv-dev libboost-all-dev libboost-iostreams-dev libcurlpp-dev libcurl4-openssl-dev protobuf-compiler libopenblas-dev libhdf5-dev libprotobuf-dev libleveldb-dev libsnappy-dev liblmdb-dev libutfcpp-dev cmake libgoogle-perftools-dev unzip python-setuptools python-dev libspdlog-dev python-six python-enum34 libarchive-dev rapidjson-dev libmapbox-variant-dev
+```
+
+On Ubuntu 18.04 or Nvidia Jetson, the network library cppnetlib is not part of the standard packages anymore and needs to be compiled beforehand:
+
+```
+wget https://github.com/cpp-netlib/cpp-netlib/archive/cpp-netlib-0.11.2-final.tar.gz
+tar xvzf cpp-netlib-0.11.2-final.tar.gz
+cd cpp-netlib-cpp-netlib-0.11.2-final
+mkdir build
+cd build
+cmake ..
+make
+sudo make install
+```
+
+With CUDA 10, a more recent version of cmake than that of Ubuntu 18.04 is required, and needs to be installed beforehand:
+
+```
+wget https://github.com/Kitware/CMake/releases/download/v3.14.0/cmake-3.14.0.tar.gz
+tar xvzf cmake-3.14.0.tar.gz
+cd cmake-3.14.0
+./bootstrap
+make
+sudo make install
```

Comment:
We should recommend the Debian packages:  https://apt.kitware.com/

Reply:
DevOps often cry when they see `sudo make install`

Reply:
OK, what do I need to change exactly ?

Reply:
```
sudo apt-get install apt-transport-https ca-certificates gnupg software-properties-common wget
wget -q -O- https://apt.kitware.com/keys/kitware-archive-latest.asc | sudo apt-key add -
sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
sudo apt-get update
sudo apt-get install cmake kitware-archive-keyring
```

Reply:
done